starter-code/apple-pie-starter-code/css/main.css

```
@@ -0,0 +1,83 @@
+body{
+	font-family: 'Montserrat', sans-serif;
+
```

Puedes remover el pequeño margen de la página blanco para que luzca más como el ejemplo
```suggestion
    margin: 0;
```